clang/tools/clang-format/ClangFormat.cpp

```diff
@@ -26,6 +26,12 @@
 #include "llvm/Support/InitLLVM.h"
 #include "llvm/Support/Process.h"
 #include <fstream>
+#include <iostream>
```

Comment:
why do you need iostream? did you have some std::cerr << debug output that you've now removed but you've left the header?